docs/user-guide/hypershift-agent-service-config.md

```diff
@@ -0,0 +1,141 @@
+# Hypershift zero worker deployment
+
+## Prerequisites
+1. Deploy hub cluster using [dev-scripts]((https://github.com/javipolo/openshift-assisted-installer-tests/tree/4515edfc9d60a587cabd3bcfdd77826cab8cc02d/ai-in-zero-worker-hypershift#deploy-an-l0-hub-cluster-using-dev-scripts))
```

Comment:
dev-scripts are mainly for development environments (i.e. not for prod). I guess the first prerequisite should be just deploying a hub cluster. So just as an example, we can indeed refer to dev-scripts.